src/darwin/Framework/CHIP/CHIPDeviceController.h

```diff
@@ -102,6 +102,11 @@ typedef void (^CHIPDeviceConnectionCallback)(CHIPDevice * _Nullable device, NSEr
  */
 - (void)shutdown;
 
+/**
+ * Converts a given X.509v3 certificate into a Matter certificate.
+ */
+- (NSData *)convertX509CertToMatterCert:(NSData *) x509Cert;
```

Comment:
This does not belong on the controller; it does not use any controller state.

Reply:
> oes not belong on the controller; it does not use any controller stat

@bzbarsky-apple : Thank you for your feedback. We provide two different platform (iOS, Android) for Application. We want to use same function on this two application. When we check Android controller, we can find below PR you already approved. so, We added this patch on iOS side also. Can you consider this is again on controller side? we need this like below patch.
- https://github.com/project-chip/connectedhomeip/pull/15551

Reply:
API design for the Android bits is up to the owners of the code (e.g. @austinh0, who reviewed that PR).

For the Darwin framework, it would be somewhat helpful to understand the use cases for this API in terms of deciding whether it should exist, where it should live, what it should be called, etc.